### PR TITLE
create convertStringToDate

### PR DESCRIPTION
**Pull request by Raymundogs**

### CODE REVIEW

File: src/main/java/core/utils/TimeUnits.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
+/**
```
Comment (edsonarios) [inline src/main/java/core/utils/TimeUnits.java:1]: In the future, I think is better use the library Calendar, for not problems with precision of the dates

**Review by edsonarios — APPROVED** [review]
Good job

File: src/main/java/core/utils/StringToDateConverter.java [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+     *
+     * @param string with the date on format like:
+     *               TODAY, YESTERDAY, TOMORROW, or
+     *               [number] [time unit] ago, or
+     *               [number] [time unit] from now, or
+     *               MM/DD/YYYY
+     * @return the date in Date type
+     */
+    public Date convertStringToDate(final String string) throws ParseException {
+        if (string == null | string.equals("")) {
```
Comment (jpgonzales27) [inline src/main/java/core/utils/StringToDateConverter.java:32]: i'm not shure about this operator | , i think you want to use an OR ||

Comment (Raymundogs) [reply src/main/java/core/utils/StringToDateConverter.java:32]: Done.

**Pull request merged**